app/entities/query.go

```diff
@@ -0,0 +1,41 @@
+package entities
+
+import (
+	"fmt"
+	"regexp"
+)
+
+// QueryParams represent request query params
+// that will be used on transport and repository level.
+type QueryParams struct {
+	Offset string
+	Limit  string
+	Sort   string
+}
+
+const (
+	numericArgsMask = `^[0-9]+$`
```

Comment:
You are making it to be too complex.
Why do you need regexp here at all?

You know that some values are valid - just compare an incoming value against that list.

Reply:
Thanks. Good point. Sometimes I am not aware of doing things hard to grasp. Will fix it ASAP